ember-cli-build.js

```diff
@@ -9,5 +9,14 @@ module.exports = function (defaults) {
     },
   });
 
-  return app.toTree();
+  const { Webpack } = require('@embroider/webpack');
+
+  return require('@embroider/compat').compatBuild(app, Webpack, {
+    staticAddonTestSupportTrees: true,
+    staticAddonTrees: true,
+    staticHelpers: true,
+    staticModifiers: true,
+    staticComponents: true,
+    splitAtRoutes: ['map'],
```

Comment:
This splits off _some_ of the depenendencies that are only used in the map route. They will be loaded when the user enters the route so the main bundle size is reduced.

Reply:
Thank you! :pray: